### PR TITLE
Created basic redirect directory structure

### PR DESCRIPTION
**Pull request by underser**
#7

### CODE REVIEW

File: modman [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ./checkout app/code/Pronko/LiqPayCheckout
 ./test app/code/Pronko/LiqPayTest
 ./admin app/code/Pronko/LiqPayAdmin
-./gateway app/code/Pronko/LiqPayGateway
\ No newline at end of file
+./gateway app/code/Pronko/LiqPayGateway
+./regirect app/code/Pronko/LiqPayRedirect
```
Comment (mcspronko) [inline modman]: Please rename regirect to redirect.

Comment (underser) [reply modman]: fixed

File: modman [hunk 2]
```diff
@@ -2,4 +2,4 @@
 ./test app/code/Pronko/LiqPayTest
 ./admin app/code/Pronko/LiqPayAdmin
 ./gateway app/code/Pronko/LiqPayGateway
-./regirect app/code/Pronko/LiqPayRedirect
\ No newline at end of file
+./redirect app/code/Pronko/LiqPayRedirect
```
Comment (mcspronko) [inline modman:5]: Thanks 👍 It seems like the directory name should also be changed to "redirect".

**Comment by mcspronko** [comment]
@underser please let me know if you need some help.

File: modman (hunk 2 above)
Comment (underser) [reply modman:5]: fixed.

**Review by mcspronko — APPROVED** [review]

**Comment by mcspronko** [comment]
Thank you @underser

**Pull request merged**